### PR TITLE
Add Media Query to Desktop version of Portfolio

### PR DESCRIPTION
**Pull request by Abdulwhbhussain**
1- Add the Necessary Background Images for Desktop Version
2- Refactoring CSS for the Desktop Version of Portfolio
3- Add Header, Work Section, About Myself, Contact Form.
4- Add Media Queries to the CSS.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1180,3 +1214,132 @@ textarea {
...
+  .headline-container1 {
+    background: url(assets/header-bg-desktop.svg);
+    position: relative;
+    width: 100vw;
+    height: 721.46px;
+    left: 34.6px;
+    top: 0;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
```
Comment (IngDavidTM) [inline style.css:1236]: - [ ] Although the background of the main action for the desktop version has been successfully changed :+1:, there are certain matters that require attention. The following steps can be taken to address them:

- The information within the section should be centered to align with the margins and padding of the template.
- The background image needs to be enlarged as it currently occupies only a portion of the background.
- The bottom left corner should have a rounded border in accordance with the template.

**Your design**

Screenshot from 2023-03-29 08-00-21

**Figma**

Screenshot from 2023-03-29 08-03-49

File: style.css [hunk 2]
```diff
@@ -1180,3 +1214,132 @@ textarea {
...
+
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
```
Comment (IngDavidTM) [inline style.css:1290]: - [ ] Your work on the card is impressive :+1:, but there are a few adjustments that need to be made to align with the Figma template. Here are the specific changes required:

- Please ensure that the project description ("A daily...") matches the width specified in the template.
- As evident from the template, there is a discrepancy in the background color of the button in comparison to yours.

- [ ] Also, while each work is divided half of it contains the project's picture and the other half contains the project's description :+1:. You should make sure that these switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth).

**Your design**

Screenshot from 2023-03-29 08-09-58

**Figma**

Screenshot from 2023-03-29 08-10-12

File: style.css [hunk 3]
```diff
@@ -1180,3 +1214,132 @@ textarea {
...
+    align-items: center;
+    padding: 114px 50px;
+    position: relative;
+    width: 100%;
+    height: 600.15px;
+    left: auto;
+    right: auto;
+    top: 0;
+  }
+}
```
Comment (IngDavidTM) [inline style.css:1345]: - [ ] Your desktop version of the about me page looks great, but it is important to make it match the Figma template as closely as possible. Please consider the following suggestions:

- The page should be divided into 2 blocks, with the left block taking up 40% and the right block taking up the remaining 60% of the page's width.
- The "Get my Resume" button needs to be redesigned to match the button design in the template.
- The languages "JavaScript", "HTML", and "CSS" should be added as shown in the Figma template.
- The top right corner should have a rounded border in accordance with the template.
**Your design**

Screenshot from 2023-03-29 08-22-29

**Figma**

Screenshot from 2023-03-29 08-22-05

File: index.html [hunk 4]
```diff
@@ -343,7 +354,7 @@ <h2 class="top-text1">About Myself</h2>
       </section>
 
       <!-- Form -->
-      <section id="contact-form" class="form-container">
+      <section id="contact" class="form-container">
         <div class="form-content-container">
           <!-- Form header -->
           <div class="form-header-container">
```
Comment (IngDavidTM) [inline index.html:360]: - [ ] Great work on creating the contact form! However, as previously mentioned, it is important for your project to match the Figma template. Please make the following changes to ensure that the contact form is aligned with the template:

- The elements of the form, including the "Get in touch" button, should be centered on the page.
- Verify that the description is also centered.
- The background image needs to be distinct from the one used in the mobile version.

**Your design**

Screenshot from 2023-03-29 08-30-49

**Figma**

Screenshot from 2023-03-29 08-30-28

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @Abdulwhbhussain 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] Your project looks impressive :+1:, but there seems to be a horizontal scroll present. It is important to address this issue to ensure that users can have a seamless experience while viewing the project. Kindly fix the horizontal scroll :+1:.

https://user-images.githubusercontent.com/105117832/228557384-f6acdeb7-c319-4d3b-9899-758f5f4fe5a0.mp4

- [x] It is important to remember that the buttons in your project should have the design interactions specified in the Figma template. This is necessary to ensure that the project meets the desired standards and delivers a consistent user experience. Please make sure that the buttons adhere to the required design interactions.

Screenshot from 2023-03-29 08-44-23


- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Abdulwhbhussain) [reply style.css]: I implemented the requested changes only the Bottom Left Corner remains in this Section.

File: style.css (hunk 3 above)
Comment (Abdulwhbhussain) [reply style.css]: i made the changes but the top right rounded corner and The languages "JavaScript", "HTML", and "CSS"  are yet to be Added.

File: index.html (hunk 4 above)
Comment (Abdulwhbhussain) [reply index.html:368]: I improved the Contact form.

**Review by braincee — APPROVED** [review]
Hi @Abdulwhbhussain  , I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**